doc/index.md

```diff
@@ -149,9 +40,6 @@ Canonical RBAC.
 
 More details are [available here](security.md).
 
-## Getting started with LXD
```

Comment:
Duplication - not needed (and I doubt anybody has a bookmark to this section, so no redirect needed).